packages/soci-snapshotter/soci-snapshotter.spec

```
@@ -69,6 +71,13 @@ Provides: %{name}(optimized-gunzip) = 0:
 %description igzip
 %{summary}.
 
+%package k8s
+Summary: Drop-in configuration for kubelet to set imageServiceEndpoint to soci-snapshotter socket
```

```suggestion
Summary: Drop-ins to override the kubelet's configurations
```

In case more soci-snapshotter overrides are provided by this sub-package other than the socket override